mono/tests/merp-json-valid.cs

```diff
@@ -103,6 +100,13 @@ public static void
 	public static void Main ()
 	{
 		JsonValidateState ();
-		JsonValidateMerp ();
+
+		var configDir = "./merp-json-valid/";
+		Directory.CreateDirectory (configDir);
```

Comment:
Probably doesn't matter yet, but I think in XA the tests can't write to the working directory so some day this may be a problem for them.

Reply:
XA won't be using this set of tests in all likelihood, as they're focusing on the merp pipeline working correctly.

That is a separate issue that may need to be fixed for the crash reporter in general though. The default `mono_crash.*.json` files made by the crash reporter are put in the working directory.

Reply:
Device crash reporting will probably need a per-platform modification, or a native callback of some sort.